doc/rest-api.md

```diff
@@ -2424,3 +2427,236 @@ Input (none at present):
+                    "config": {
+                        "source": "/var/lib/lxd/storage-pools/default"
+                    },
+                    "driver": "dir",
+                    "used_by": null
+                }
+            ],
+            "networks": [
+                {
+                    "name": "lxdbr0",
+                    "description": "",
+                    "type": "bridge",
+                    "config": {
+                        "ipv4.address": "10.8.219.1/24",
+                        "ipv4.nat": "true",
+                        "ipv6.address": "fd42:f5a2:e47e:2185::1/64",
+                        "ipv6.nat": "true"
+                    },
+                    "used_by": null,
+                    "managed": true
+                }
+            ]
+    	}
+    }
+
+### DELETE
+ * Description: disable clustering
+ * Introduced: with API extension `clustering`
+ * Authentication: trusted
+ * Operation: sync
+ * Return: standard return value or standard error
+
+Input (none at present):
+
+    {
+    }
+
+## `/1.0/cluster/nodes`
+### GET
+ * Description: list of LXD nodes in the cluster
+ * Introduced: with API extension `clustering`
+ * Authentication: trusted
+ * Operation: sync
+ * Return: list of dicts with information about each node
+
+	{
+		"type": "sync",
+		"status": "Success",
+		"status_code": 200,
+		"operation": "",
+		"error_code": 0,
+		"error": "",
+		"metadata": [
+			{
+				"name": "lxd1",
+				"url": "https://10.1.1.101:8443",
+				"database": true,
+				"state": "ONLINE"
+			},
+			{
+				"name": "lxd2",
+				"url": "https://10.1.1.102:8443",
+				"database": true,
+				"state": "ONLINE"
+			},
+		]
+	} 
+
+### POST
+ * Description: bootstrap, join, or accept a node in the cluster
+ * Introduced: with API extension `clustering`
+ * Authentication: trusted or untrusted
+ * Operation: sync or async
+ * Return: various payloads depending on the input
+
+Input (bootstrap a new cluster):
+
+    {
+		"name": "lxd1",
+	}
+
+Return background operation or standard error.
+
+Input (request to join an existing cluster):
+
+	{
+		"name": "node2",
+		"target_address": "10.1.1.101:8443",
+		"target_cert": "-----BEGIN CERTIFICATE-----MIFf\n-----END CERTIFICATE-----",
+		"target_password": "sekret"
+	}
+
+Return background operation or standard error.
+
+Input (accept a node requesting to join the cluster):
+
+	{
+		"name": "node2",
+		"address": "10.1.1.102:8443",
+		"schema": 2,
+		"api": 63,
+		"target_password": "sekret"
+	}
+
+Return information about raft nodes in the cluster and the private key
+of the cluster certificate:
+
+	{
+		"type": "sync",
+		"status": "Success",
+		"status_code": 200,
+		"operation": "",
+		"error_code": 0,
+		"error": "",
+		"metadata": {
+			"raft_nodes": [
+				{
+					"id": 1,
+					"address": "10.1.1.101:8443"
+				},
+				{
+					"id": 2,
+					"address": "10.1.1.102:8443"
+				}
+			],
+			"private_key": "LS0tLS1CRU"
+		}
+	}
+
+## `/1.0/cluster/nodes/<name>`
```

Comment:
What's going on with this one? Looks like some bad copy/paste from the storage volume one.

Reply:
I'm not sure what you mean. I did take storage as template, but the info here is just documenting input/output formats for POST /1.0/cluster/members